library/src/main/java/com/novoda/simplechromecustomtabs/SimpleChromeCustomTabs.java

```diff
@@ -109,15 +109,24 @@ public boolean isConnected() {
     }
 
     /**
-     * Starts a new session for Chrome Custom Tabs usage. Can be used to warmup particular Urls.
+     * Tells SimpleChromeCustomTabs that a potential Url might be launched. This will do pre DNS resolution that will speed things up.
+     * @param uri
+     * @return WebNavigator with warmed up url
```

Comment:
You mention that this will return a WebNavigation, but this method in fact returns `void`